### PR TITLE
Fixes #183, Sherpa/Romeo gray error

### LINKED ISSUE #183: Fix Sherpa/Romeo gray.submission error and contribute to DSpace
When a journal's policies are unknown to SHERPA / RoMEO, the Mirage2 theme classifies it as "gray." However, there are no gray messages in messages.xml and no gray class defined in the Mirage2 CSS. Good candidate for contributing to DSpace core when fixed.

### PR DESCRIPTION
**Pull request by amandafrench**
Also adds the /submission/submission.xsl to the vtmirage2 theme. Can you please review, @AmberPoo1? @alawvt needn't do all the code review!

### CODE REVIEW

File: dspace/modules/xmlui-mirage2/src/main/webapp/themes/vtmirage2/xsl/aspect/submission/submission.xsl [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                <xsl:if test="contains(@rend,'blue')">
+                    <xsl:text>label label-info</xsl:text>
+                </xsl:if>
+                <xsl:if test="contains(@rend,'yellow')">
+                    <xsl:text>label label-warning</xsl:text>
+                </xsl:if>
+                <xsl:if test="contains(@rend,'green')">
+                    <xsl:text>label label-success</xsl:text>
+                </xsl:if>
+                <xsl:if test="contains(@rend,'gray')">
```
Comment (amandafrench) [inline dspace/modules/xmlui-mirage2/src/main/webapp/themes/vtmirage2/xsl/aspect/submission/submission.xsl]: Here's where the style is added.

File: dspace/modules/xmlui-mirage2/src/main/webapp/themes/vtmirage2/xsl/aspect/submission/submission.xsl [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    <xsl:text>label label-default</xsl:text>
+                </xsl:if>                
+
```
Comment (amandafrench) [inline dspace/modules/xmlui-mirage2/src/main/webapp/themes/vtmirage2/xsl/aspect/submission/submission.xsl:110]: Those lines above are where the "gray" style is added to the message.

**Comment by alawvt** [comment]
Please test with a RoMEO ungraded journal such as _Victorian Poetry_ (ISSN=0042-5206).

**Comment by AmberPoo1** [comment]
@alawvt, @amandafrench , sure, thanks.

**Comment by AmberPoo1** [comment]
@amandafrench Great! Now ungraded journal shows publisher information correctly. 
screen shot 2016-02-08 at 5 38 58 pm

**Pull request merged**